#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00000001055055ac, pid=15888, tid=26883
#
# JRE version: OpenJDK Runtime Environment Zulu15.40+19-CA (15.0.7+4) (build 15.0.7+4-MTS)
# Java VM: OpenJDK 64-Bit Server VM Zulu15.40+19-CA (15.0.7+4-MTS, mixed mode, tiered, compressed oops, g1 gc, bsd-aarch64)
# Problematic frame:
# C  [Python3+0x1495ac]  collect+0x200
#
# No core dump will be written. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# If you would like to submit a bug report, please visit:
#   http://www.azul.com/support/
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#

---------------  S U M M A R Y ------------

Command Line: -Dfile.encoding=UTF8 -ea -Xmx1024m 

Host: MacBookPro18,3 arm64 1 MHz, 8 cores, 16G, Darwin 21.3.0
Time: Mon Aug  1 16:09:57 2022 KST elapsed time: 0.100074 seconds (0d 0h 0m 0s)

---------------  T H R E A D  ---------------

Current thread (0x000000012d6c8200):  JavaThread "Python Reference Queue" daemon [_thread_in_native, id=26883, stack(0x0000000296ee8000,0x00000002970eb000)]

Stack: [0x0000000296ee8000,0x00000002970eb000],  sp=0x00000002970ea790,  free space=2057k
Native frames: (J=compiled Java code, A=aot compiled Java code, j=interpreted, Vv=VM code, C=native code)
C  [Python3+0x1495ac]  collect+0x200
C  [Python3+0x149310]  PyGC_Collect+0x78
C  [_jpype.cpython-38-darwin.so+0x1c5dc]  JPGarbageCollection::triggered()+0x38
C  [_jpype.cpython-38-darwin.so+0x28e10]  Java_org_jpype_ref_JPypeReferenceNative_wake+0x1c
j  org.jpype.ref.JPypeReferenceNative.wake()V+0
j  org.jpype.ref.JPypeReferenceQueue$Worker.run()V+42
j  java.lang.Thread.run()V+11 java.base@15.0.7
v  ~StubRoutines::call_stub
V  [libjvm.dylib+0x3e00a4]  JavaCalls::call_helper(JavaValue*, methodHandle const&, JavaCallArguments*, Thread*)+0x324
V  [libjvm.dylib+0x3df17c]  JavaCalls::call_virtual(JavaValue*, Klass*, Symbol*, Symbol*, JavaCallArguments*, Thread*)+0xf8
V  [libjvm.dylib+0x3df244]  JavaCalls::call_virtual(JavaValue*, Handle, Klass*, Symbol*, Symbol*, Thread*)+0x64
V  [libjvm.dylib+0x48e83c]  thread_entry(JavaThread*, Thread*)+0xbc
V  [libjvm.dylib+0x8cb660]  JavaThread::thread_main_inner()+0xa0
V  [libjvm.dylib+0x8c8b5c]  Thread::call_run()+0x88
V  [libjvm.dylib+0x711b78]  thread_native_entry(Thread*)+0x1d0
C  [libsystem_pthread.dylib+0x7240]  _pthread_start+0x94

Java frames: (J=compiled Java code, j=interpreted, Vv=VM code)
j  org.jpype.ref.JPypeReferenceNative.wake()V+0
j  org.jpype.ref.JPypeReferenceQueue$Worker.run()V+42
j  java.lang.Thread.run()V+11 java.base@15.0.7
v  ~StubRoutines::call_stub

siginfo: si_signo: 11 (SIGSEGV), si_code: 2 (SEGV_ACCERR), si_addr: 0x00000001694c9ee8

Register to memory mapping:

 x0=0x0000000175c38630 points into unknown readable memory: 0x0000000105650960 | 60 09 65 05 01 00 00 00
 x1=0x00000002970ea8f8 is pointing into the stack for thread: 0x000000012d6c8200
 x2=0x00000002970ea8f0 is pointing into the stack for thread: 0x000000012d6c8200
 x3=0x0 is NULL
 x4=0x0000000000008903 is an unknown value
 x5=0x0 is NULL
 x6=0x0000000000000001 is an unknown value
 x7=0x00000000004c4b40 is an unknown value
 x8=